cmds/exp/console/console.go

```diff
@@ -47,27 +47,7 @@ func main() {
 		libinit.CreateRootfs()
 	}
 
-	in, out := io.Reader(os.Stdin), io.Writer(os.Stdout)
-
-	// This switch is kind of hokey, true, but it's also quite convenient for users.
-	switch {
-	// A raw IO port for serial console
-	case []byte(*serial)[0] == '0':
-		u, err := openUART(*serial)
-		if err != nil {
-			log.Fatalf("Console exits: sorry, can't get a uart: %v", err)
-		}
-		in, out = u, u
-	case *serial == "i8042":
-		u, err := openi8042()
-		if err != nil {
-			log.Fatalf("Console exits: sorry, can't get an i8042: %v", err)
-		}
-		in, out = u, os.Stdout
-	case *serial == "stdio":
-	default:
-		log.Fatalf("Console exits: console must be one of stdio, i8042, or an IO port with a leading 0 (e.g. 0x3f8)")
-	}
+	in, out, err := console(*serial)
```

Comment:
This is an ineffectual assign. err is reassigned immediately by p.Start, so err is unused in this assignment.

Reply:
yeah.

Reply:
fixed.